src/app/core/entity-components/entity-form/entity-form/entity-form.component.ts

```diff
@@ -72,16 +75,40 @@ export class EntityFormComponent<T extends Entity = Entity> implements OnInit {
 
   form: EntityForm<T>;
 
+  private saveInProgress = false;
+
   constructor(
     private entityFormService: EntityFormService,
-    private alertService: AlertService
+    private alertService: AlertService,
+    private entityMapper: EntityMapperService,
+    private confirmationDialog: ConfirmationDialogService
   ) {}
 
   ngOnInit() {
     this.buildFormConfig();
     if (this.editing) {
       this.switchEdit();
     }
+    this.entityMapper
+      .receiveUpdates(this.entity.getConstructor())
+      .pipe(filter(({ entity }) => entity.getId() === this.entity.getId()))
+      .subscribe(({ entity }) => this.applyChanges(entity));
+  }
+
+  private async applyChanges(entity) {
+    if (this.saveInProgress) {
+      // this is the component that currently saves the values -> no need to apply changes.
+      return;
+    }
+    if (
+      this.form.pristine ||
+      (await this.confirmationDialog.getConfirmation(
+        $localize`Load changes?`,
+        $localize`Local changes are in conflict with updated values. Do you want the local changes to be overwritten with the latest values?`
```

Comment:
```suggestion
        $localize`Local changes are in conflict with updated values synced from the server. Do you want the local changes to be overwritten with the latest values?`
```

Reply:
Not necessarily. the same issue happens, if a person was editing two forms at the same time but only saves one of them.

Reply:
true. But that case currently gives a very confusing UX anyway: the popup for the second tab will show immediately after saving the first tab (while you still see the first tab). So nobody apart from us two will understand which changes the question exactly relates to :see_no_evil: